### PR TITLE
Fix #588: Refactor 'Review' to 'Revision'

### LINKED ISSUE #588: Review should be updated to Revision
As per new mocks, Topic Review Tab has been updated to Topic Revision Tab,

Screenshot 2020-01-03 at 12 11 29 PM

So, all files related to that needs to be updated like package name, file names, test cases names, xml files names, variable names, strings.xml, drawables, etc.

In short refactor `Review` with `Revision`.

### PR DESCRIPTION
**Pull request by tanhengyeow**
## Explanation

Fixes #588

All files affected e.g. package name, file names, test cases names, xml files names, variable names, strings.xml, drawables, etc are refactored.

## Checklist

- [X] The PR title starts with "Fix #bugnum: ", followed by a short, clear summary of the changes. (If this PR fixes part of an issue, prefix the title with "Fix part of #bugnum: ...".)
- [X] The PR explanation includes the words "Fixes #bugnum: ..." (or "Fixes part of #bugnum" if the PR only partially fixes an issue).
- [X] The PR follows the [style guide](https://github.com/oppia/oppia-android/wiki/Coding-style-guide).
- [X] The PR does not contain any unnecessary auto-generated code from Android Studio.
- [X] The PR is made from a branch that's **not** called "develop".
- [X] The PR is made from a branch that is up-to-date with "develop".
- [X] The PR's branch is based on "develop" and not on any other branch.
- [ ] The PR is **assigned** to an appropriate reviewer in both the **Assignees** and the **Reviewers** sections.

### CODE REVIEW

File: app/src/main/java/org/oppia/app/topic/revision/RevisionSkillSelector.kt [hunk 1]
```diff
@@ -0,0 +1,8 @@
+package org.oppia.app.topic.revision
+
+import org.oppia.app.model.SkillSummary
+
+/** Listener for when a skill is selected for review. */
```
Comment (nikitamarysolomanpvt) [inline app/src/main/java/org/oppia/app/topic/revision/RevisionSkillSelector.kt:5]: Rename review to revision in comment.

**Comment by nikitamarysolomanpvt** [comment]
Please include the screen shot of UI updated.

**Review by nikitamarysolomanpvt — CHANGES_REQUESTED** [review]
I believe  we need to refactor review card to revision card , term review in Topic proto i.e. in Domain module, in JSON file (not so sure), and several comments in protos (i.e. EphemeralQuestion,Question(not so sure))@rt4914  Please finalise the requirement and suggest and deferring to you.

**Review by rt4914 — CHANGES_REQUESTED** [review]
This looks great. But there are some places in which we have missed this. Mainly `domain` and `model` layer. So I suggest you press `Shift` key twice and global search for `Review` keyword and that way you can easily find where this word is getting used and we can replace that. 
First try to work using this approach, if this does not work out, I can mention the files in which you need to make changes.

**Comment by rt4914** [comment]
@veena14cs Keep track of this PR, as this PR and your subtopic ReviewCard PR will have merge conflicts depending upon who's PR gets merged first.

**Comment by rt4914** [comment]
> I believe we need to refactor review card to revision card , term review in Topic proto i.e. in Domain module, in JSON file (not so sure), and several comments in protos (i.e. EphemeralQuestion,Question(not so sure))@rt4914 Please finalise the requirement and suggest and deferring to you.

Thanks a lot @nikitamarysolomanpvt . I have replied to @tanhengyeow about the approach.

**Comment by rt4914** [comment]
Closing this PR because of inactivity.

**Pull request closed without merging**